tron/commands/display.py

```diff
@@ -394,22 +392,38 @@ class DisplayEvents(TableDisplay):
     resize_fields = ['entity']
 
 
-def display_node(source):
+def display_node(source, _=None):
+    if not source:
```

Comment:
Since the source has been checked for multiple display function, it might be worth to have a function for it.

Reply:
i'd rather not make the diff even larger in less relevant parts of the codebase. this display functionality needs to be refactored anyway.